WcaOnRails/db/migrate/20220516124717_add_team_for_wat.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddTeamForWat < ActiveRecord::Migration[6.1]
+  def change
+    Team.create(friendly_id: 'wat', email: "archive@worldcubeassociation.org")
```

Comment:
Please create in the initial migration as `hidden: true`, similar to https://github.com/thewca/worldcubeassociation.org/pull/6900/files#diff-5d6622d6d46140624e4da2f52867db2f36420549b50a2ca2defdf0fb8377a846R5

Reply:
done